### PR TITLE
update CSS board 2023-2024

### PR DESCRIPTION
**Pull request by BorhanSaflo**
### What are you trying to accomplish?

Update the CSS board for 2023-2024

### How are you accomplishing it?

By updating the meetTheBoard.md

### Is there anything reviewers should know?

N/A

### Checks before you create your pull request

-   [x] Did you read and follow article requirements?
-   [x] Did you run prettier?
-   [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

File: css/meetTheBoard.md [hunk 1]
```diff
@@ -5,34 +5,34 @@ sidebar_label: Meet the Board
...
-**Event Coordinators** - Julia Sabelli | sabell11@uwindsor.ca, Eva Mudgal | mudgal@uwindsor.ca, Nick Boisclair | boiscla@uwindsor.ca, Musaib Nagani | naganim@uwindsor.ca  
-**Secretary** - Abbie Dewhirst | dyck41@uwindsor.ca
+**Merchandising Coordinator** - Calder Johnson | johnso1x@uwindsor.ca  
+**Promotions Coordinator** - Hiba Rehman | rehman72@uwindsor.ca  
+**Career Advisor** - Borhan Saflo | saflob@uwindsor.ca  
+**Senior Representative** - Drini Husi | husid@uwindsor.ca  
+**Second Year Representative** - Norika Upadhyay | upadhy82@uwindsor.ca  
+**First Year Representatives** - TBD Fall 2023  
+**Event Organizer** - Nafiz Hasan | hasan55@uwindsor.ca  
+**Event Coordinators** - Anika Khan | khan661@uwindsor.ca, Akshat Sharma | sharma8d@uwindsor.ca, Ahmad Munim | munima@uwindsor.ca, Mathew | pellarim@uwindsor.ca
```
Comment (jere-mie) [inline css/meetTheBoard.md]: you didn't include mat's last name

**Review by jere-mie — CHANGES_REQUESTED** [review]
one small change

File: css/meetTheBoard.md (hunk 1 above)
Comment (BorhanSaflo) [reply css/meetTheBoard.md]: Fixed, thank you.

**Review by jere-mie — APPROVED** [review]
LGTM 🐧🐧

**Pull request merged**